ci-operator/config/cloud-bulldozer/e2e-benchmarking/cloud-bulldozer-e2e-benchmarking-master.yaml

```diff
@@ -271,6 +274,7 @@ tests:
       COMPUTE_NODE_TYPE: m6g.xlarge
       CONTROL_PLANE_INSTANCE_TYPE: m6g.xlarge
       ENABLE_LAYER_3: "false"
+      FEATURE: NetworkSegmentation
```

Comment:
Is the feature needed on OCP 4.18?

Reply:
Only for UDN builds, something similar to what we do here:
https://github.com/openshift/release/blob/master/ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__aws-4.18-nightly-x86.yaml#L297-L305